Windows gonna be windows

JorisGoosen opened the pull request:
- should allow for local characters in windows
-- Fixes https://github.com/jasp-stats/jasp-issues/issues/633
- fixes small mistake with version of module not being correct
- Adds a test to testmodule for encoding of string in data
- Also changed unicode characters in common.R to \unicode escape chars notation
Double check for COPY_CRYPT
- fixes https://github.com/jasp-stats/INTERNAL-jasp/issues/780

boutinb reviewed (approved):
I think this is ok... I have tested it only on Mac, and it seems to work.
But this is a quite a risky change just before the release. I will ask some people on mac/windows/linux tomorrow to test the nightly -built version with the 'strange characters' file.

JorisGoosen commented:
Yeah I agree it is a bit risky, but I've tested it myself on windows and linux and the application works better then before. Aka, you can actually filter on unicode.

I'll just revert the whitespace change in rbridge.cpp and then Ill merge it.

JorisGoosen committed:
Windows gonna be windows
- should allow for local characters in windows
-- Fixes https://github.com/jasp-stats/jasp-issues/issues/633
- fixes small mistake with version of module not being correct
- Adds a test to testmodule for encoding of string in data
- Also changed unicode characters in common.R to \unicode escape chars notation
Double check for COPY_CRYPT
- fixes https://github.com/jasp-stats/INTERNAL-jasp/issues/780
